src/main/java/com/krafit/financialcalcs/LoanCalculatorApplication/LoanCalculatorHelper.java

```diff
@@ -0,0 +1,14 @@
+package com.krafit.financialcalcs.LoanCalculatorApplication;
```

Comment:
change to ExpectedPersonalLoanEMICalculationResponse

Reply:
changed to ExpectedPersonalLoanEMICalculationResponse